### ISSUE TITLE
Add cleanup function(s)

### ISSUE THREAD

**Issue by NilsIrl** [glacambre/firenvim#318]
Add a cleanup function that does the opposite of `firenvim#install()`.

**Comment by glacambre** [comment]
Good idea! I'll implement that tomorrow or the day after :).

Are you trying to uninstall Firenvim? If yes, could you describe why?

**Comment by NilsIrl** [comment]
> Are you trying to uninstall Firenvim?

Nope and yes.

I was trying to uninstall it because the post-update hook for minpac didn't run and I had to run `firenvim#install(0)` manually. So I wanted to clean everything so I could find a way for the post-update hook to run correctly without me having to do stuff manually.

**Comment by Th3Whit3Wolf** [comment]
Have the instructions been tested?
[We were unable to use opt and do together](https://github.com/fannheyward/coc-rust-analyzer/issues/90)

**Comment by NilsIrl** [comment]
@Th3Whit3Wolf, your on the wrong issue which confused me for a second. But yes it does work for me.

Your issue is a bit different. I don't see why yarn wasn't able to run on your side since it's not a vim command. So it's the kind of cases where it should "just" work.

**Comment by Th3Whit3Wolf** [comment]
Interesting. I tested this with [vim-clap](https://github.com/liuchengxu/vim-clap)

```vim
call minpac#add('liuchengxu/vim-clap', {'do': 'packad vim-clap | call clap#helper#build_all()'})
```

and errors for me.

**Comment by NilsIrl** [comment]
I don't know if it is related but there is a typo in the thing you linked: ~~packad~~ packadd

Also the type has to be `opt`.

**Comment by glacambre** [comment]
Closed in #324 :)

**Comment by Th3Whit3Wolf** [comment]
@NilsIrl you're right that fixed it.
